### DISCUSSION TITLE
Successfully migrated Flowise to Bun

### DISCUSSION THREAD

**Discussion by jitsmaster** [FlowiseAI/Flowise#1867] [Ideas]
Hi folks,

I successfully converted Flowise from Node to Bun. The result is simply amazing.

The UI load up flows less than 200ms. Chat delay reduced up to 60%. Also, just installing and update packages is 70% faster.

It will be great if I can create a PR and allow Flowise to switch to using Bun. Please let me know if that is OK and Flowise team does intend to migrate to Bun at some point.

Arnold

**Comment by HenryHengZJ** [comment]
wow! definitely create a draft PR (we'll definitely take a look!)

**Reply by jitsmaster** [reply]
Will do. Sorry for the late response. Really busy recently
